README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Bookstore: connect to API
-In this project, I connected React-Redux application to an existing API to create and remove books in a remote server.
+# Bookstore: styling
+In this project, I styled the App using Zeplin as a collaboration tool.
```

Comment:
- [ ] Please add the "Getting Started" section to your README as shown in this [template](https://github.com/microverseinc/readme-template). You have already given a description of how to get started, but you need to ensure that the **Getting Started** header is there. This is a general requirement 😎 
- [ ] I noticed that your project does not have a license and, because of that, the link to the `MIT` license in your README is broken. Please fix this as it is essential that every software (open source included) has a license 👍